src/traces/scattergl/index.js

```diff
@@ -259,15 +259,28 @@ function sceneUpdate(gd, subplot) {
         // draw traces in proper order
         scene.draw = function draw() {
             var i;
+            var allReadyDrawLines = Lib.repeat(false, scene.count);
+            var allReadyDrawMarkers = Lib.repeat(false, scene.count);
```

Comment:
Do we still need this extra logic now that https://github.com/plotly/plotly.js/pull/2990 is merged?

Reply:
IN https://codepen.io/erwanmas/pen/JaymVr
or 
https://github.com/plotly/plotly.js/pull/2979/files#diff-2f69844ce1835e61b2c9811d852a7dd9

look for the lines of the green trace 

It's clearly visible when you don't use the default colors ... because default colors have a alpha channel .